src/Symfony/Component/Yaml/Tests/InlineTest.php

```diff
@@ -563,9 +563,10 @@ public function getTestsForDump()
     /**
      * @dataProvider getTimestampTests
      */
-    public function testParseTimestampAsUnixTimestampByDefault(string $yaml, int $year, int $month, int $day, int $hour, int $minute, int $second)
+    public function testParseTimestampAsUnixTimestampByDefault(string $yaml, int $year, int $month, int $day, int $hour, int $minute, int $second, int $microsecond)
     {
-        $this->assertSame(gmmktime($hour, $minute, $second, $month, $day, $year), Inline::parse($yaml));
+        $expectedDate = (new \DateTimeImmutable($yaml))->format('U');
+        $this->assertSame(($microsecond) ? (float) "$expectedDate.$microsecond" : (int) $expectedDate, Inline::parse($yaml));
```

Comment:
Parenthesis are not needed.

```suggestion
        $this->assertSame($microsecond ? (float) "$expectedDate.$microsecond" : (int) $expectedDate, Inline::parse($yaml));
```